lib/capistrano/tasks/database.rake

```diff
@@ -0,0 +1,109 @@
+namespace :forkcms do
+  namespace :database do
+
+    backup_file = 'mysql_backup.sql'
+
+    desc 'Create a backup of the database'
+    task :backup do
+      on roles(:web) do
+        # Fetch our flags to reach the database
+        flags = get_mysql_connection_flags
+
+        # Abort if no flags are found
+        if flags == nil
+          next
+        end
+
+        execute :mysqldump, "#{flags} --skip-lock-tables > #{release_path}/#{backup_file}"
+      end
+    end
+
+    desc 'Restore the database from a backup file'
+    task :restore do
+      on roles(:web) do
+        # Check if the file exists.
```

Comment:
useless comment

Reply:
28a9b3

Reply:
well, since I'm not familiar with Ruby and this gem is intended to be used by people who are not familiar with Ruby, I'd leave the comments. It doesn't hurt for them to be there

Reply:
In my opinion the raised error explains it very nicely.

Reply:
10cc3f9